inst/lintr/linters.csv

```diff
@@ -59,7 +59,7 @@ missing_package_linter,robustness common_mistakes
 namespace_linter,correctness robustness configurable executing
 nested_ifelse_linter,efficiency readability
 nested_pipe_linter,readability consistency configurable
-no_tab_linter,style consistency deprecated
+no_tab_linter,defunct
```

Comment:
Is the defunct tag documented + auto-excluded like deprecated?

Reply:
It's not given its own page. I think the idea is to hide these linters as much as possible.

https://github.com/r-lib/lintr/blob/15b75d80dfc932bbd32fdf08a0d7ed9f5b38a540/R/linter_tags.R#L9-L12